ha/fault_tolerance/fault_tolerance_driver.py

```diff
@@ -41,42 +37,29 @@ def __init__(self, wait_time=10):
         """Init method"""
         self._wait_time = wait_time
         ConfigManager.init("fault_tolerance")
-        self._consumer = self._get_consumer()
+        self.node_fault_monitor = NodeFaultMonitor()
+        self.cluster_stop_monitor = ClusterStopMonitor()
 
-    def _get_consumer(self) -> MessageBusConsumer:
+    def start(self):
         """
-           Returns an object of MessageBusConsumer class which will listen on
-           cluster_event message type and callback will be executed
+        start the threads
         """
-        self._consumer_id = Conf.get(const.HA_GLOBAL_INDEX, f'FAULT_TOLERANCE{_DELIM}consumer_id')
-        self._consumer_group = Conf.get(const.HA_GLOBAL_INDEX, f'FAULT_TOLERANCE{_DELIM}consumer_group')
-        self._message_type = Conf.get(const.HA_GLOBAL_INDEX, f'FAULT_TOLERANCE{_DELIM}message_type')
-        return MessageBus.get_consumer(consumer_id=self._consumer_id, \
-                                consumer_group=self._consumer_group, \
-                                message_type=self._message_type, \
-                                callback=self.process_message)
-
-    def process_message(self, message: str):
-        """Callback method for MessageConsumer"""
-        Log.debug(f'Received the message from message bus: {message}')
-        try:
-            EventAnalyzer(message.decode('utf-8'))
-        except Exception as err:
-            Log.error(f'Failed to analyze the event: {err}')
-            return CONSUMER_STATUS.FAILED
-        return CONSUMER_STATUS.SUCCESS
+        self.node_fault_monitor.start()
+        self.cluster_stop_monitor.start()
 
     def poll(self):
-        """Contineously polls for message bus for k8s_event message type"""
+        Log.debug("FaultTolerance poll")
```

Comment:
similarly here there should be function docstring. Right?

Reply:
Good catch. Looks like the docstring moved one line down

Reply:
Ahh ok..In that PR also, this was missed. I checked only comment is there or not but not the alignment. Anyways, thanks for fixing.